### PR TITLE
(Doc) 1.21.0 Regular doc update

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: ik673626 <igor.kazmyr@broadcom.com>

## Proposed changes

Update to the Readme (the release notes section) and Changelog for 1.21.0

## Release Notes

Milestone:

Changelog: 1.21.0

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by lauren-li — CHANGES_REQUESTED** [review]
Thanks @IgorCATech for putting this together! I made a couple suggestions to the Readme, which can also be applied to the Changelog.

Additionally, I think we can add items for some of the other PRs we merged since the last release (including for the ZE FTP extension). Please check my reply to your squad message for more details on this request. Thanks again!

File: packages/zowe-explorer/README.md [hunk 1]
```diff
@@ -23,31 +23,29 @@ More information:
...
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.20.0
+## What's new in Zowe Explorer 1.21.0
 
 **Added**:
 
-- Added a Github action bot that automates the issue triage.
+- Add a progress bar for the simultaneous deletion of multiple jobs
```
[the remark this answers is not in the document]
Comment (IgorCATech) [reply packages/zowe-explorer/README.md:38]: Actually, we use the Past tense for such notes in this doc. You can check the previously added notes

File: packages/zowe-explorer/README.md [hunk 2]
```diff
@@ -23,31 +23,29 @@ More information:
...
-- Update to use Visual Studio Code's standard confirmation prompt for the Data Sets, USS, and Job trees when clicking on a Favorited profile that does not exist.
-- Updated the deletion prompt for the USS and Jobs trees.
-- Updated the placeholder text in the `Add Profile` entry field
+- Changed the `DataSet uploaded successfully` message type. Now messages are shown in the status bar instead of the notification pop-up
 
 **Fixed**:
 
-- Fixed the Not Found issue that resulted from attempts to delete a member whose parent data set was already deleted using multi-delete.
+- Fixed the issue that caused JCL errors in the JOBS tree to be displayed as `undefined:undefined(undefined)`
+- Fixed the Theia input box issue
```
[the remark this answers is not in the document]
Comment (IgorCATech) [reply packages/zowe-explorer/README.md]: addressed

File: packages/zowe-explorer-ftp-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## `1.21.0`
+
+- Added unit tests for the FTP USS functionality [#1582](https://github.com/zowe/vscode-extension-for-zowe/pull/1582) and [1596](https://github.com/zowe/vscode-extension-for-zowe/pull/1596). Thanks @tiantn and JillieBeanSim
+- Added the new logger API. You can now use the Zowe Explorer API logger functions to standardize messages and specify message severity [#1518](https://github.com/zowe/vscode-extension-for-zowe/pull/1518). Thanks @katelynienaber
```
Comment (lauren-li) [inline packages/zowe-explorer-ftp-extension/CHANGELOG.md:6]: This changelog item is accurate if the target audience is developers of the ZE FTP extension.

However, if we are addressing end users of the ZE FTP extension, this item might be a bit confusing. The new logger API is not something that the end user will be touching directly. Rather, they will just see that the ZE FTP extension's messages are standardized and have severity specified. I would suggest rewording to something like:
> Standardized user interface and log messages with severity specifications using Zowe Explorer API’s logger functions.

File: packages/zowe-explorer/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
...
+- Updated dependencies for improved security [#1591](https://github.com/zowe/vscode-extension-for-zowe/pull/1591) and [#1601](https://github.com/zowe/vscode-extension-for-zowe/pull/1601). Thanks @lauren-li
+- Updated Theia tests to use the latest Theia version [#1566](https://github.com/zowe/vscode-extension-for-zowe/pull/1566). Thanks @JillieBeanSim
+- Fixed the issue that caused JCL errors in the JOBS tree to be displayed as `undefined:undefined(undefined)` [#1584](https://github.com/zowe/vscode-extension-for-zowe/pull/1584). Thanks @roman-kupriyanov
+- Fixed the Theia input box issue that caused entered values to be incorrectly validated [#1580](https://github.com/zowe/vscode-extension-for-zowe/pull/1580). Thanks @JillieBeanSim
+- Fixed the issue that caused the removal of unsaved credentials of a profile in the Jobs tree after deleting a job. Now when you delete a job from the Jobs tree with a profile that does not have the stored credentials, the profile keeps the cached credentials [#1449](https://github.com/zowe/vscode-extension-for-zowe/pull/1449). Thanks @JillieBeanSim
```
Comment (crawr) [inline packages/zowe-explorer/CHANGELOG.md:14]: This is thanks to Nikolay

Comment (IgorCATech) [reply packages/zowe-explorer/CHANGELOG.md]: Addressed

Comment (lauren-li) [reply packages/zowe-explorer/CHANGELOG.md]: @IgorCATech For consistency, I updated the linked reference for this item to PR #1524, which is the PR that fixed issue #1449, as all the other changelog items link to PRs rather than issues.

**Review by lauren-li — APPROVED** [review]
This PR looks good to me! Thanks @IgorCATech for putting this together and working with me on the refinements!

**Review by crawr — APPROVED** [review]

File: packages/zowe-explorer/CHANGELOG.md (hunk 4 above)
Comment (IgorCATech) [reply packages/zowe-explorer/CHANGELOG.md]: Good catch! Thank you for your vigilance and attention to detail
